client/state/data-layer/wpcom-http/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { get, noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import { WPCOM_HTTP_REQUEST } from 'state/action-types';
+import { extendAction } from 'state/utils';
+
+/**
+ * Returns the appropriate fetcher in wpcom given the request method
+ *
+ * fetcherMap :: String -> (Params -> Query -> [Body] -> Promise)
+ *
+ * @param {string} method name of HTTP method for request
+ * @returns {Function} the fetcher
+ */
+const fetcherMap = method => get( {
```

Comment:
Thanks for answering @retrofox! That's interesting and odd as it seems like it gives us assumptions we think we should be able to rely on but then breaks them.

In the end, it's all in the documentation, but I would assume that this should remove a post.

`wpcom.req.del( '/sites/1337/posts/18' )`

Though this is all just a side conversation for another time and place 😊